frontend/admin-ui/package.json

```diff
@@ -21,6 +21,7 @@
     "node-fetch": "^2.6.0",
     "quasar": "^1.0.0",
     "vue-apollo": "^3.0.0-rc.6",
+    "vue-client-only": "^2.0.0",
```

Comment:
That package is necessary in order to work with components that are built client-side only. I'm not sure yet of the relevance of that, but I know the map needs to be client only.